components/data/limma_voom/requirements.txt

```diff
@@ -0,0 +1,2 @@
+maayanlab-bioinformatics @ git+https://github.com/Maayanlab/maayanlab-bioinformatics
```

Comment:
rpy2 is an optional dependency of this package but required for limma voom, please add it to the requirements.txt